packages/map/index.html

```diff
@@ -20,7 +20,7 @@
     <!-- DEFAULT EXAMPLES -->
     <!-- <div class="react-container" data-config="/examples/private/nhis.json"></div> -->
```

Comment:
remove this file from your git history

Reply:
Reset file to dev here:
https://github.com/CDCgov/cdc-open-viz/pull/2031/commits/35f88f470e235c4988b35116dd078b0dfdb72dd0